packages/autometrics/src/index.ts

```diff
@@ -1,2 +1,6 @@
-export { setMetricsExporter } from "./instrumentation";
-export * from "./wrappers";
+import startTSPlugin from "@autometrics/typescript-plugin";
+
+module.exports = startTSPlugin;
```

Comment:
It would be good to add some comments here explaining the reasons for the different types of exports